static/_redirects

```diff
@@ -183,6 +183,10 @@
 /guides/launch/run-agent/ /guides/launch/setup-agent-advanced/ 301
 /guides/launch/sagemaker/ /guides/launch/setup-launch-sagemaker/ 301
 /guides/launch/vertex/ /guides/launch/setup-vertex/ 301
+/ref/python/launch-library/ /launch-library/ 301
```

Comment:
Based on the above comment, might need to update redirect path to include "guides"

Reply:
If all of the pages still exist but just under the new path, then use a splat in place of all of these rules, down in the bottom section.

```
/guides/launch/* /launch-library/:splat 301
```

Reply:
You should always use splats if possible, with specific rules that should override the splat (because the source or destination page is different, for example) coming first. There are some times when we should have used splats before (like these) but that I wasn't sure about when I converted most of the redundant rules to splats. Cloudflare has a pretty conservative limit on the # of individual rules (I think it may be 100 or 1000).

Reply:
In https://github.com/wandb/docs/pull/1373#discussion_r2130577037 there are two different kinds of splats. In the first example, any request to a page in `/guides/launch/` will redirect to the same page name under `/launch-library/`. In the second example, any request to a page in `/guides/launch/` will redirect to exactly the page `/launch-library/` (the landing page, in this case).